spec/msteams_hermes/message_spec.rb

```diff
@@ -6,6 +6,12 @@
 require "msteams_hermes/style"
 
 RSpec.describe MsTeamsHermes::Message do
+  let(:fact) { { title: "foo", value: "bar" } }
+  let(:fact_set) { MsTeamsHermes::Components::FactSet.new(facts: [fact]) }
+  let(:card) { MsTeamsHermes::Components::AdaptiveCard.new(body: [fact_set]) }
+
+  let(:webhook_url) { "https://xingag.webhook.office.com/webhookb2/some-more-path-tokens" }
```

Comment:
Do we want that URL to appear? I mean it is not containing any secret, but `xingag` might be interesting to know, for "people" interested.

Reply:
Is better to use a generic reference indeed 🙏 
At the moment though the gem is under `xing` organisation 😃